test_accounts/verification_test_accounts.md

```diff
@@ -2,36 +2,48 @@
 
 These test users have a variety of synthetic VA records such as service history and disability rating attached to them and can be used to test the [Veteran Verification API](https://developer.va.gov/explore/verification/docs/veteran_verification?version=current).
 
-To log in to the sandbox environment with a test account, make sure you:
+**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
+
+## Logging in with ID.me
+
+To log in to the sandbox environment using ID.me, make sure you:
+
+  1. Choose an account that is specific to the API you want to test.
+  2. Select **ID.me** to sign in to the sandbox environment.
+  3. Enter the ID.me username and password (**Password1234!**).
```

Comment:
Just to clarify what the password is for ID.me test accounts, I would have a separate sentence that states what the password is. The parenthesis could be misinterpreted as part of the password.
```suggestion
  3. Enter the ID.me username and password. The password for all ID.me test accounts is: **Password1234!**
```